stdlib/pwd.pyi

```diff
@@ -8,6 +8,12 @@ class struct_passwd(Tuple[str, str, int, int, str, str, str]):
     pw_gecos: str
     pw_dir: str
     pw_shell: str
+    @property
+    def n_fields(self) -> int: ...
+    @property
+    def n_sequence_fields(self) -> int: ...
+    @property
+    def n_unnamed_fields(self) -> int: ...
```

Comment:
There are actually class properties:

```python
srittau@blitz:~$ python3
Python 3.7.3 (default, Jan 22 2021, 20:04:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pwd
>>> pwd.struct_passwd.n_fields
7
>>> 
```

Reply:
Maybe:
```suggestion
    n_fields: ClassVar[int]
    n_sequence_fields: ClassVar[int]
    n_unnamed_fields: ClassVar[int]
```

You will need to add `from typing import ClassVar`.